packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -9,6 +9,42 @@ import { SegmentioSettings } from './index'
 import { version } from '../../generated/version'
 import { getAvailableStorageOptions, UniversalStorage } from '../../core/user'
 
+declare global {
```

Comment:
Couple of nit picks
1. Can we encapsulate this into a separate file/folder, maybe under core/client-hints/...? 

2. I don't think we want to declare a global; it will leak into anyone who imports our lib, so it can conflict with their typings when later versions of typescript support this out of the box, or if they have custom typings. It would actually better to just do const getUserAgentDate = (): UserAgentData =>(navigator as any).userAgentData